### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by MAliNaqvi**
latest version of docker doesn't contain "Client"

https://github.com/docker/docker-py/blob/master/docker/__init__.py#L2

### CODE REVIEW

File: hyper_sh/__init__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import json
 import os
 
-from docker import Client as DockerClient
+from docker import APIClient as DockerClient
```
Comment (tardyp) [inline hyper_sh/__init__.py]: I guess we need to try both imports.

Comment (MAliNaqvi) [reply hyper_sh/__init__.py]: @tardyp I am not clear on why the travis build is failing. is it using an older docker version?

should it be:

```
try:
    from docker import Client as DockerClient
except ImportError:
    from docker import APIClient as DockerClient
```
even though the latest docker-py doesn't use `Client`?

Comment (tardyp) [reply hyper_sh/__init__.py]: usually you do the other way around.
try import new API, and then import old api

**Comment by MAliNaqvi** [comment]
@tardyp I have updated the imports

**Comment by conornash** [comment]
Ping - is this ok to merge?

**Pull request merged**